test/reflection.jl

```diff
@@ -553,3 +553,11 @@ let
     @test length(a) == 0
     @test length(b) == 0
 end
+
+# Adds test for Issue #15924
+let
+    a = @code_lowered sin(1.0)
+    @test isa(a, LambdaInfo)
+    b = Base.uncompressed_ast(a)
+    @test length(b) == 4
```

Comment:
I don't think we should test this. There's no reason this IR has to have length 4; it could easily change due to small changes in the compiler or implementation of the function.